src/requirementslib/models/url.py

```diff
@@ -132,7 +132,7 @@ def _parse_fragment(self):
         extras = self.extras
         for q in fragments:
             key, _, val = q.partition("=")
-            val = unquote_plus(val.replace("+", " "))
```

Comment:
This replacement is not necessary as `unquote_plus` will handle plus characters.

Reply:
yeah, i wasn't thinking too hard at the time probably...